src/crons/cron.ts

```diff
@@ -0,0 +1,16 @@
+import { Bindings } from "../bindings";
+import { ExecutionContext } from "hono/dist/types/context";
+import delete_old_data_cron from "./delete_old_data_cron";
+
+async function cron(event: ScheduledEvent, env: Bindings, context: ExecutionContext) {
+  switch (event.cron) {
+    case env.DELETE_OLD_DATA_CRON:
```

Comment:
what happens if this environment variable is not set for some reason?

Reply:
The cron won't be triggered; hence, I configured it through the wrangler.toml file.